lib/livebook_web/live/modal_component.ex

```diff
@@ -4,7 +4,7 @@ defmodule LivebookWeb.ModalComponent do
   @impl true
   def render(assigns) do
     ~L"""
-    <div class="fixed z-40 inset-0"
+    <div class="fixed z-1100 inset-0"
```

Comment:
Tailwind now has a just-in-time compiler and makes it possible to generate classes with hard-coded values, which is neat for one-time use cases like this :)

```suggestion
    <div class="fixed z-[10000] inset-0"
```